GPA.cpp

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+    const int numCourses = 3; 
+    double totalGradePoints = 0.0;
+    double totalCreditHours = 0.0;
+    double gradePoints = 0.0;
+    double grade1, grade2, grade3;
+    int credits1, credits2, credits3;
+    
+    
+    // Course 1 User Input
+    cout << "Enter the grade (0.0 - 4.0) for Course 1: ";
```

Comment:
This is using the wrong data type and calculation

Reply:
ok thank you

Reply:
correct